src/main/java/uk/gov/hmcts/reform/ucmc/enums/AllocatedTrack.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.ucmc.enums;
+
+import uk.gov.hmcts.reform.ucmc.model.ClaimValue;
+
+public enum AllocatedTrack {
+    SMALL_CLAIM,
+    FAST_CLAIM,
+    MULTI_CLAIM;
+
+    //TODO: logic for when no claim value -> checking with Darren and Luke
+    public static AllocatedTrack getAllocatedTrack(ClaimValue claimValue, ClaimType claimType) {
```

Comment:
Should we move this utility method out of this enum, and put into a separate component/service?